includes/class-activitypub.php

```diff
@@ -478,4 +510,17 @@ public static function user_register( $user_id ) {
 			$user->add_cap( 'activitypub' );
 		}
 	}
+
+	/**
+	 * Filter the query to only show the current author's extra fields
+	 *
+	 * @param WP_Query $query The WP_Query instance (passed by reference).
+	 *
+	 * @return void
+	 */
+	public static function filter_get_posts_query( $query ) {
+		if ( $query->get( 'post_type' ) === 'ap_extrafield' ) {
+			$query->set( 'author', get_current_user_id() );
```

Comment:
I'm trying to understand why we need this? The `author` field is already specified in the only two `WP_Query` instances trying to get this data.

Reply:
This shows only the extra field of a user and hides the extra fields from the others in the wp-admin pages.

Reply:
Ok! Then it would probably be good to either 1) check that `author` isn't already set before setting it, or 2) only add this filter on that page? Again, not that important, we can revisit if it's a problem.

Reply:
Makes sense! I moved the filters around a bit.